dataikuapi/dss/savedmodel.py

```diff
@@ -69,6 +67,23 @@ def set_active_version(self, version_id):
         self.client._perform_empty(
             "POST", "/projects/%s/savedmodels/%s/versions/%s/actions/setActive" % (self.project_key, self.sm_id, version_id))
 
+    def delete_versions(self, versions, remove_intermediate=True):
+        """
+        Delete version(s) of the saved model
+
+        :param versions: list of versions to delete
+        :type versions: list[str]
+        :param remove_intermediate: also remove intermediate versions (default: True). In the case of a partitioned
+        model, an intermediate version is created every time a partition has finished training.
+        :type remove_intermediate: bool
+        """
+        body = {
+            "versions": versions,
```

Comment:
Initial remark stands then: should auto-wrap versions if isn't an array.

Reply:
I do not think so. It is dealt with server-side, by the custom deserializer.

Reply:
ok, so, following our discussion:
- the server side deserializer is dumber (but less complex)
- the ability to use a string instead of an array for a single version is handled in the wrapper, but not advertised.